--- source/haproxy-config.template	2022-03-03 10:54:24.319868575 -0500
+++ haproxy-config.template	2022-03-03 10:53:04.226643114 -0500
@@ -174,6 +174,7 @@
 
   {{ if .BindPorts -}}
 frontend public
+  http-response del-header Set-Cookie2
     {{ if eq "v4v6" $router_ip_v4_v6_mode }}
   bind :{{ env "ROUTER_SERVICE_HTTP_PORT" "80" }}
   bind :::{{ env "ROUTER_SERVICE_HTTP_PORT" "80" }} v6only
@@ -225,6 +226,7 @@
 # determined by the next backend in the chain which may be an app backend (passthrough termination) or a backend
 # that terminates encryption in this router (edge)
 frontend public_ssl
+  http-response del-header Set-Cookie2
     {{- if ne (env "ROUTER_SYSLOG_ADDRESS") "" }}
   option tcplog
     {{- end }}
@@ -268,6 +270,7 @@
   server fe_sni 127.0.0.1:{{ env "ROUTER_SERVICE_SNI_PORT" "10444" }} weight 1 send-proxy
 
 frontend fe_sni
+  http-response del-header Set-Cookie2
   # terminate ssl on edge
   bind 127.0.0.1:{{ env "ROUTER_SERVICE_SNI_PORT" "10444" }} ssl
   {{- if isTrue (env "ROUTER_STRICT_SNI") }} strict-sni {{ end }}
@@ -357,6 +360,7 @@
   server fe_no_sni 127.0.0.1:{{ env "ROUTER_SERVICE_NO_SNI_PORT" "10443" }} weight 1 send-proxy
 
 frontend fe_no_sni
+  http-response del-header Set-Cookie2
   # terminate ssl on edge
   bind 127.0.0.1:{{ env "ROUTER_SERVICE_NO_SNI_PORT" "10443" }} ssl crt {{ firstMatch ".+" .DefaultCertificate "/var/lib/haproxy/conf/default_pub_keys.pem" }} accept-proxy
     {{- with (env "ROUTER_MUTUAL_TLS_AUTH") }}
